cluster/images/tools.cmds

```diff
@@ -3,29 +3,35 @@ sudo gsutil -m -o 'GSUtil:parallel_thread_count=1' -o 'GSUtil:sliced_object_down
 
 # The below tarball is a capture of the result of running this on a live instance:
 # curl -L https://raw.githubusercontent.com/yyuu/pyenv-installer/master/bin/pyenv-installer | sudo bash
-# Doing that every time blindly is not compatible with a secure environment
+# Doing that every time blindly is insecure
 sudo rm -rf /root/.pyenv
 sudo tar xvzf /opt/tools/python/pyenv.tar.gz -C /root
+sudo chmod a+x /tmp/mk_python_venv
 
 sudo /root/.pyenv/bin/pyenv install 3.6.10
 sudo /root/.pyenv/versions/3.6.10/bin/pip install --upgrade pip
-sudo chmod a+x /tmp/mk_python_venv
-sudo /tmp/mk_python_venv peach 1.6 3.6.10
 sudo /tmp/mk_python_venv peach 1.7 3.6.10
 
-sudo mkdir -p /opt/tools/cuppa-chart/1.8
-sudo unzip /opt/tools/cuppa/1.8/cuppa.jar cuppa-chart/* -d /tmp/cuppa-chart-1.8
-sudo mv /tmp/cuppa-chart-1.8/cuppa-chart/* /opt/tools/cuppa-chart/1.8/
-sudo /tmp/mk_python_venv --no-extract-tarball cuppa-chart 1.8 3.6.10
+# Shared for new Cuppa apps
+sudo dpkg -i /opt/tools/python/libffi6_3.2.1-9_amd64.deb
+sudo dpkg -i /opt/tools/python/libffi-dev_3.2.1-9_amd64.deb
+sudo /root/.pyenv/bin/pyenv install 3.9.4
+sudo /root/.pyenv/versions/3.9.4/bin/pip install --upgrade pip
 
-sudo mkdir -p /opt/tools/cuppa-chart/1.8.1
-sudo unzip /opt/tools/cuppa/1.8.1/cuppa.jar cuppa-chart/* -d /tmp/cuppa-chart-1.8.1
-sudo mv /tmp/cuppa-chart-1.8.1/cuppa-chart/* /opt/tools/cuppa-chart/1.8.1/
-sudo /tmp/mk_python_venv --no-extract-tarball cuppa-chart 1.8.1 3.6.10
+# Pycuppa only
+sudo mkdir -p /opt/tools/pycuppa/2.0rc/
+sudo unzip /opt/tools/cuppa/2.0rc/cuppa.jar pycuppa/* -d /tmp/pycuppa-2.0rc
+sudo mv /tmp/pycuppa-2.0rc/pycuppa/* /opt/tools/pycuppa/2.0rc
+sudo /tmp/mk_python_venv --no-extract-tarball pycuppa 2.0rc 3.9.4
 
+# Cuppa-chart only
+sudo mkdir -p /opt/tools/cuppa-chart/2.0rc
+sudo unzip /opt/tools/cuppa/2.0rc/cuppa.jar cuppa-chart/* -d /tmp/cuppa-chart-2.0rc
+sudo mv /tmp/cuppa-chart-2.0rc/cuppa-chart/* /opt/tools/cuppa-chart/2.0rc
+sudo /tmp/mk_python_venv --no-extract-tarball cuppa-chart 2.0rc 3.9.4
```

Comment:
I think we can remove the cuppa-chart environment since cuppa-chart is no longer called in pipeline5